src/backends/sentry_backend_inproc.c

```diff
@@ -212,8 +227,12 @@ make_signal_event(
     sentry_value_t frames = sentry__value_new_list_with_size(frame_count);
     for (size_t i = 0; i < frame_count; i++) {
         sentry_value_t frame = sentry_value_new_object();
-        sentry_value_set_by_key(frame, "instruction_addr",
-            sentry__value_new_addr((uint64_t)backtrace[frame_count - i - 1]));
+        void *ip = backtrace[frame_count - i - 1];
+#if defined SENTRY_PLATFORM_ANDROID
+        sentry__symbolize(ip, fill_frame, &frame);
```

Comment:
I don’t think you need these changes here.
We already have an option to symbolicate on-device here:

https://github.com/getsentry/sentry-native/blob/bac5d366593e68f240b4446c3bf1ab1d1556a7bd/include/sentry.h#L789-L798
That option is on-by-default on android.